conan/tools/cmake/cmake.py

```diff
@@ -83,8 +83,13 @@ def configure(self, variables=None, build_script_folder=None):
             pkg_folder = self._conanfile.package_folder.replace("\\", "/")
             arg_list.append('-DCMAKE_INSTALL_PREFIX="{}"'.format(pkg_folder))
         if platform.system() == "Windows" and self._generator == "MinGW Makefiles":
-            # It seems this doesn't work in the toolchain file, it needs to be here in command line
+            # It seems these don't work in the toolchain file, they need to be here in command line
```

Comment:
I wonder if this is because the toolchain doesnt set the variables with "CACHE" status?

Reply:
no, already tried.